test/RibbonTreasuryVault.ts

```diff
@@ -97,6 +103,44 @@ describe("RibbonTreasuryVault", () => {
     minDeposit: parseUnits("1", 18),
     availableChains: [CHAINID.ETH_MAINNET],
   });
+
+  behavesLikeRibbonOptionsVault({
+    name: `Ribbon BAL Treasury Vault (Call)`,
+    tokenName: "Ribbon BAL Treasury Vault",
+    tokenSymbol: "rBAL-TSRY",
+    asset: BAL_ADDRESS[chainId],
+    assetContractName: "IWETH",
+    strikeAsset: USDC_ADDRESS[chainId],
+    collateralAsset: BAL_ADDRESS[chainId],
+    chainlinkPricer: CHAINLINK_BAL_PRICER[chainId],
+    deltaStep: BigNumber.from(STRIKE_STEP.BAL),
+    depositAmount: parseEther("20"),
+    minimumSupply: BigNumber.from("10").pow("10").toString(),
+    expectedMintAmount: BigNumber.from("2000000000"),
```

Comment:
think current `Error: VM Exception while processing transaction: reverted with reason string 'limit price not better than mimimal offer'` error is because block is on friday (not thursday)